pkg/glusterutils/brick_status_gd2.go

```diff
@@ -0,0 +1,30 @@
+package glusterutils
+
+// VolumeBrickStatus gets brick status info from glusterd2 using rest api
+func (g GD2) VolumeBrickStatus(vol string) ([]BrickStatus, error) {
+	client, err := initRESTClient(g.config)
+	if err != nil {
+		return nil, err
+	}
+	brickstatusinfo, err := client.BricksStatus(vol)
+	if err != nil {
+		return nil, err
+	}
+	brickstatus := make([]BrickStatus, len(brickstatusinfo))
+	for idx, info := range brickstatusinfo {
+		status := BrickStatus{
+			Hostname: info.Info.Hostname,
+			PeerID:   info.Info.PeerID.String(),
+			PID:      info.Pid,
+			Path:     info.Info.Path,
+			Volume:   vol,
+		}
+		if info.Online {
+			status.Status = 1
```

Comment:
can we give `status` a better name something link `BrickStatusObj` or something?

status.Status Someone might get confused in understanding.

Reply:
Ack. Will rename this as `brickStatusObj`